test/e2e/generate_kube_test.go

```diff
@@ -207,4 +207,19 @@ var _ = Describe("Podman generate kube", func() {
 		Expect(psOut).To(ContainSubstring("test1"))
 		Expect(psOut).To(ContainSubstring("test2"))
 	})
+
+	It("podman generate kube with volume", func() {
+		vol1 := filepath.Join(podmanTest.TempDir, "vol-test1")
+		err := os.MkdirAll(vol1, 0755)
+		Expect(err).To(BeNil())
+
+		session1 := podmanTest.Podman([]string{"run", "-d", "--pod", "new:test1", "-v", vol1 + ":/volume/:z", "alpine", "top"})
+		session1.WaitWithDefaultTimeout()
+		Expect(session1.ExitCode()).To(Equal(0))
+
+		kube := podmanTest.Podman([]string{"generate", "kube", "test1"})
```

Comment:
Can we play the container we generate here with `play kube` and verify it has a volume present?

Reply:
fixed!